graph/graph.go

```diff
@@ -93,8 +99,21 @@ func (g *Graph) GetParent(id string) interface{} {
 			break
 		}
 	}
+	return parentID
+}
 
-	return g.Get(parentID)
+// IsNibling checks to see if second is the child of a sibling of the first.
+func (g *Graph) IsNibling(fst, snd string) bool {
+	if snd == "." || snd == "" {
+		return false
+	}
+	if fst == g.GetParentID(snd) {
+		return false
+	}
+	if AreSiblingIDs(fst, snd) {
```

Comment:
yeah, I can do that, it was just easier to re-use the logic from `GetParentID` and I thought that would be sufficient since `GetParentID` is only looking at the edges and not doing string calculations so we're only really using the string value for the terminal case.